<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnterWirelessSettingPage</class>
 <widget class="QWidget" name="EnterWirelessSettingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">EnterWirelessSettingPage{background-color:transparent;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="m_groupBox">
     <property name="title">
      <string>Wireless Setting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="m_enterSettingPromptLB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Enter the wireless settings for XAVN2001:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="WirelessSettingPannel" name="m_wirelessSettingPane" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="WirelessSettingOperatePane" name="m_operatePane" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SearchRouterPane" name="m_searchRouterPane" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WirelessSettingPannel</class>
   <extends>QWidget</extends>
   <header>wirelesssettingpannel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WirelessSettingOperatePane</class>
   <extends>QWidget</extends>
   <header>wirelesssettingoperatepane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchRouterPane</class>
   <extends>QWidget</extends>
   <header>searchrouterpane.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
